ecmascript/transforms/classes/src/super_field.rs

```diff
@@ -193,6 +193,36 @@ impl<'a> Fold for SuperCalleeFolder<'a> {
                     right,
                 })
             }
+
+            Expr::Call(CallExpr {
+                callee: ExprOrSuper::Expr(ref expr),
+                ref args,
+                ref type_args,
+                ..
+            }) => match &**expr {
+                Expr::Member(MemberExpr {
+                    obj: ExprOrSuper::Super(Super { span: super_token }),
+                    ..
+                }) => {
+                    let this_arg = ThisExpr {
```

Comment:
I looked at the diffs from test and this seems to be the cause. Maybe you need to use aliased variable instead.

Reply:
Although I use 
``` rust
let this_arg = ThisExpr { span: DUMMY_SPAN };
```
the `this` still be transformed into `_this`

Reply:
I mean the reverse. Instead of using `ThisExpr`, you may try injecting `var _this = this`.